docs/data/charts/scatter/scatter.md

```diff
@@ -105,7 +105,17 @@ See [Axis—Grid](/x/react-charts/axis/#grid) documentation for more information
 
 {{"demo": "GridDemo.js"}}
 
-### CSS 🚧
+### CSS
+
+You can target scatter elements with CSS selector by using the following data attributes:
+
+- `data-series-id='<series id>'`
+- `data-highlighted=true|false`
+- `data-faded=true|false`
```

Comment:
Not sure if this looks better:

```suggestion
- `data-highlighted=true` or `data-highlighted=false`
- `data-faded=true` or `data-faded=false`
```

The previous option didn't look that great to me: 

![image](https://github.com/user-attachments/assets/b0660eb6-cb02-49b9-b6ab-0b4d386fdd3a)